test/models/tf_ov_model_zoo/run_infer_multiple.sh

```diff
@@ -46,6 +67,18 @@ while read -r line; do
     (( $retcode == 1 )) && failed_models+=("${args[0]}")
 done < "$MANIFEST"
 
+if [ "$BENCHMARK" == "YES" ] && [ -f "$CSVFILE" ]; then
```

Comment:
Not directly related to this PR, but in terms of ease-of-use, "-b" would be better for benchmark mode than an unqualified "YES" or "NO".

Reply:
Yes, but keeping the argument parsing simple in .sh script with mandatory value: -b YES